sympy/solvers/solveset.py

```diff
@@ -892,10 +893,14 @@ def solveset(f, symbol=None):
         return solve_univariate_inequality(f, symbol, relational=False)
 
     if isinstance(f, (Expr, Number)):
-        if real is True:
+        if domain is S.Reals or real:
```

Comment:
As per @hargup 's [comment](https://github.com/sympy/sympy/pull/9500#discussion_r35492301) on this. I think that `real` should be removed. I am just guessing. So `real` be removed from `(symbol.is_real is True)`.